packages/ra-ui-materialui/src/layout/Theme/ThemesContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import { RaThemeOptions } from './types';
+
+export const ThemesContext = createContext<ThemesContextValue>({});
+
+export interface ThemesContextValue {
+    darkTheme?: RaThemeOptions;
+    lightTheme?: RaThemeOptions;
+    defaultToLightTheme?: boolean;
```

Comment:
Why no a `defaultTheme` instead, allowing you to default to the dark theme if you want?

Reply:
👍 You're right, it's better. Added in the next commit.